src/VerifyU2FController.js

```diff
@@ -69,25 +84,60 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
       className: 'verify-u2f-form',
       noCancelButton: true,
       save: _.partial(Okta.loc, 'verify.u2f.retry', 'login'),
+      noButtonBar: function () {
+        return !window.hasOwnProperty('u2f');
+      },
       modelEvents: {
         'error': '_showRetry',
         'request': '_hideRetry'
       },
 
-      formChildren: [
-        FormType.View({
-          View: '\
+      formChildren: function () {
+        var result = [];
+
+        if (!window.hasOwnProperty('u2f')){
+          if(BrowserFeatures.isFirefox()){
```

Comment:
nit: this might be easier to read if we do something like
```
var errorCode;
 if(BrowserFeatures.isFirefox()){
  errorCode = 'u2f.error.noFirefoxExtension';
} else if (this.options.appState.get('factors').length === 1) {
  errorCode='verify.u2f.error.notSupportedBrowser.oneFactor';
}
```
and then add the ```HtmlErrorMessageView```